src/tests/doubles/stubs/emails.py

```
@@ -0,0 +1,44 @@
+from notifications import EmailSenderInterface
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)